README.md

```diff
@@ -1,31 +1,153 @@
 # WheatFspm
+WheatFspm is a Functional Structural Plant Model (FSPM) of wheat based on an innovative modelling framework for a better understanding of grass morphogenesis and C-N economy in interaction with environmental conditions
 
-WheatFspm contains a set of sub-packages (named in git submodules):
-* CN-Wheat: 
-* Elong Wheat: 
-* ...
+![alt text](_media/Vegetative_stages_topview.gif "Growing canopy")
+# Description
+WheatFspm consists in a set of sub-models (named submodules in git) which share inputs/outputs through an MTG object:
 
-## Developers
+![alt text](_media/Modular_structure.png "WheatFSPM workflow") 
+*Adapted from Gauthier et al. (2020)*
 
-This  package contains [submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) ([in French](https://git-scm.com/book/fr/v2/Utilitaires-Git-Sous-modules)).
+* *Farquhar-Wheat*: Farquhar-based model of photosynthesis, stomatal conductance, organ temperature and transpiration.
+* *Elong-Wheat*: regulation of leaf and internode elongation by C and N metabolites, temperature and coordination rules.
+* *Growth-Wheat*: growth in biomass of leaves, internodes and roots ; related consumption in C and N metabolites.
+* *CN-Wheat*: synthesis and degradation of C and N metabolites at organ level and allocation between tillers' organs.
+* *Respi-Wheat*: respiratory-costs related to the main biological processes.
+* *Senesc-Wheat*: organ senescence and consequences in organ biomass, green area and remoblisation of C and N metabolites.
+* *Fspm-Wheat*: interfaces (facades) for reading/updating information between each sub-model and the MTG. Also includes the scripts to be run for using all sub-models.
+
+
+# Table of Contents
+- [WheatFspm](#wheatfspm)
+- [Description](#description)
+- [Table of Contents](#table-of-contents)
+- [Installation](#installation)
+  * [Prerequisites](#prerequisites)
+  * [Installing](#installing)
+    + [Users](#users)
+    + [Developers](#developers)
+- [Usage](#usage)
+  * [NEMA](#nema)
+  * [Papier_FSPMA2016](#papier-fspma2016)
```

Comment:
Papier -> Paper

Reply:
Unfortunately, the directory was actually named that way